src/main/kotlin/org/rust/ide/intentions/JoinWildcardsIntention.kt

```diff
@@ -0,0 +1,38 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.intentions
+
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiElement
+import org.rust.lang.core.psi.RsPatTupleStruct
+import org.rust.lang.core.psi.RsPatWild
+import org.rust.lang.core.psi.ext.ancestorStrict
+import org.rust.lang.core.psi.ext.endOffset
+import org.rust.lang.core.psi.ext.startOffset
+
+class JoinWildcardsIntention : RsElementBaseIntentionAction<List<RsPatWild>>() {
+    override fun getFamilyName(): String = "Join wildcards into `..`"
+    override fun getText(): String = familyName
+
+    override fun findApplicableContext(project: Project, editor: Editor, element: PsiElement): List<RsPatWild>? {
+        val structPat = element.ancestorStrict<RsPatTupleStruct>() ?: return null
```

Comment:
I think that `RsPatTup`s can be also applicable for this intention (for example `let (a, ..) = (1, 2, 3);`).

Reply:
And apparently slices, as well. I didn't even know you could match slices - TIL.

Reply:
Yeah I find that very common, working on the plugin gives me a lot of new knowledge about Rust :D